sonaranalyzer-dotnet/src/Rspec2Resx.targets

```diff
@@ -18,7 +18,6 @@
     <PropertyGroup>
       <PowerShellExe Condition="'$(PowerShellExe)'==''">%WINDIR%\System32\WindowsPowerShell\v1.0\powershell.exe</PowerShellExe>
     </PropertyGroup>
-    <Exec Command="$(PowerShellExe) -command &quot;if ((Get-ExecutionPolicy) -ne &apos;Unrestricted&apos;) {echo &apos;Powershell: error 0000: You have to run Set-ExecutionPolicy Unrestricted (In powershell32)&apos;} &quot; " />
     <Exec ConsoleToMsBuild="true"
       Command="$(PowerShellExe) -NonInteractive -ExecutionPolicy Unrestricted -command &quot;  &amp; { ..\..\..\scripts\rspec\rspec2resx.ps1 $(Rspec2Resx)  $(ProjectDir)../../rspec/$(Rspec2Resx) $(ProjectDir)RspecStrings.resx } &quot;"/>
```

Comment:
Even if you have removed the previous line, I'd expect this line to fail now as you don't have the right.

Reply:
locally I'm having failures if I don't delete the lines...

Reply:
Interesting, what's your local policy level? Could you try a `git clean -dfx` and then rebuild to see if everything is working as expected?

Reply:
```
PS C:\Workspace\sonar-dotnet> get-executionpolicy
Unrestricted
PS C:\Workspace\sonar-dotnet> .\scripts\build\dev-build.ps1 -restore -build -metadata

Build FAILED.

Powershell : error 0000: You have to run Set-ExecutionPolicy Unrestricted (In powershell32) [C:\Workspace\sonar-dotnet\sonaranalyzer-dotnet\src\SonarAnalyzer.VisualBasic\SonarAnalyzer.VisualBasic.csproj]
```

Reply:
and w/o the line it works